source/bson-corpus/bson-corpus.rst

```diff
@@ -338,6 +338,27 @@ Implementations MAY ignore or modify them to match legacy treatment of
 deprecated types.  The ``converted_bson`` and ``converted_extjson`` fields MAY
 be used to test conversion to a standard type or MAY be ignored.
 
+Prose Tests
+===========
+
+The following tests have not yet been automated, but MUST still be tested.
+
+1. Prohibit null bytes in null-terminated strings when encoding BSON
+--------------------------------------------------------------------
+
+The BSON spec uses null-terminated strings to represent document field names and
+regex components (i.e. pattern and flags/options). Drivers MUST assert that null
+bytes are prohibited in the following contexts when encoding BSON.
+
+* Field name within a root document
```

Comment:
There is no BSON data for these prose tests. The purpose of the test is to assert that the driver does not allow raw BSON to be produced where document fields (or the regex components) contain null bytes. In some drivers, this could be asserting that a Document class prohibits such a string its argument. In the case of libbson, it might be asserting that the API to encode a document field does not allow this (whether by raising an error or simply not allowing it at all because the field is received as a null-terminated string w/o a corresponding length).

Are you suggesting hex strings for the `char *` payload that includes a null byte?